app/api/datastore/entities/v1/crime_application.rb

```diff
@@ -8,7 +8,7 @@ class CrimeApplication < BaseApplicationEntity
 
         expose :returned_at, expose_nil: false
         expose :return_details, expose_nil: false
-        expose :supporting_evidence
+        expose :supporting_evidence, expose_nil: false
```

Comment:
Would it make more sense to return empty array in the method we are using?

```ruby
        def supporting_evidence
          submitted_value('supporting_evidence')
        end
```

Reply:
good idea will sort